plugins/Coala/Coala.py

```diff
@@ -0,0 +1,20 @@
+import re
+import random
+
+from errbot import BotPlugin, re_botcmd
+
+class Coala(BotPlugin):
+
+    def callback_message(self, msg):
+        """
+        Shout at the user for using an upercased c in coala.
+        """
+        emots = [":(", ":angry:", ":anger:", ":confounded:",
+                 ":disappointed:", ":triumph:"]
+        
+        if re.search(r'C+[oO]+[aA]+[lL]+[aA]+', msg.body):
```

Comment:
We don't want to respond on small `c` :/ ? This is the regex used currently in cobot